src/test-utils/msw-handlers.ts

```diff
@@ -0,0 +1,49 @@
+import { http, HttpResponse } from 'msw'
+
+export const handlers = [
+  // Mock organization API
+  http.get('/api/organizations', () => {
+    return HttpResponse.json({
+      data: [
+        { id: 'org-1', name: 'Acme Corp', logo: '/logo1.png' },
+        { id: 'org-2', name: 'Beta Inc', logo: '/logo2.png' },
+      ]
+    })
+  }),
+
+  // Mock coaching sessions API
+  http.get('/api/coaching_sessions/:id', ({ params }) => {
+    return HttpResponse.json({
+      data: {
+        id: params.id,
+        title: 'Session #1',
+        coaching_relationship_id: 'rel-1',
+        scheduled_date: '2025-07-04T10:00:00Z',
+      }
+    })
+  }),
+
+  // Mock coaching relationships API
+  http.get('/api/organizations/:orgId/coaching_relationships', () => {
```

Comment:
```suggestion
  http.get('/api/organizations/:organizationId/coaching_relationships', () => {
```

I think? For consistency